packages/pn-personafisica-webapp/src/api/delegations/__test__/Delegations.api.test.ts

```diff
@@ -69,55 +68,39 @@ describe('Delegations api tests', () => {
   });
 
   it('revokes a delegation', async () => {
-    const mock = new MockAdapter(apiClient);
-    mock.onPatch(REVOKE_DELEGATION('7')).reply(204);
+    mockApi(apiClient, 'PATCH', REVOKE_DELEGATION('7'), 204, undefined);
     const res = await DelegationsApi.revokeDelegation('7');
     expect(res).toStrictEqual({ id: '7' });
-    mock.reset();
-    mock.restore();
   });
 
   it("doesn't revoke a delegation", async () => {
-    const mock = new MockAdapter(apiClient);
-    mock.onPatch(REVOKE_DELEGATION('10')).reply(200);
+    mockApi(apiClient, 'PATCH', REVOKE_DELEGATION('10'), 200, undefined);
```

Comment:
remove useless undefined here and after

Reply:
changes made into: [430db73](https://github.com/pagopa/pn-frontend/pull/938/commits/430db73dd4e6ee0215846db2b30db4717bdd1763)